src/main/java/com/echobox/api/linkedin/types/posts/TargetEntity.java

```diff
@@ -0,0 +1,120 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.echobox.api.linkedin.types.posts;
+
+import com.echobox.api.linkedin.jsonmapper.LinkedIn;
+import com.echobox.api.linkedin.types.StaffCountRanges;
+import com.echobox.api.linkedin.types.objectype.Locale;
+import com.echobox.api.linkedin.types.urn.URN;
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.List;
+
+/**
+ * Target Entity
+ * @see <a href="https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/shares/posts-api#targetentities">Target Entities</a>
+ *
+ * @author Sergio Abplanalp
+ */
+public class TargetEntity {
+  // CPD-OFF
```

Comment:
Would it be better to set the CPD-OFF in the older ones so it gets automatically removed when the old API is removed?

Reply:
Good suggestion, will make the change